README.rst

```diff
@@ -188,7 +188,7 @@ module.
 
     logging.basicConfig(level=logging.INFO)
     l = logging.getLogger('fluent.test')
-    h = handler.FluentHandler('app.follow', host='host', port=24224)
+    h = handler.FluentHandler('app.follow', host='host', port=24224, buffer_overflow_handler=handler)
```

Comment:
I sent pull request to avoid variable name confliction by changing variable name in example:
https://github.com/fluent/fluent-logger-python/pull/87